commit 46e624b95c36d729bdf24010fff11d16f6fe94fa
Author: Gleb Natapov <gleb@redhat.com>
Date:   Mon Aug 24 11:54:20 2009 +0300

    KVM: Change irq routing table to use gsi indexed array
    
    Use gsi indexed array instead of scanning all entries on each interrupt
    injection.
    
    Signed-off-by: Gleb Natapov <gleb@redhat.com>
    Signed-off-by: Avi Kivity <avi@redhat.com>

diff --git a/virt/kvm/kvm_main.c b/virt/kvm/kvm_main.c
index 22b520b54411..3bee94892774 100644
--- a/virt/kvm/kvm_main.c
+++ b/virt/kvm/kvm_main.c
@@ -957,7 +957,6 @@ static struct kvm *kvm_create_vm(void)
 	if (IS_ERR(kvm))
 		goto out;
 #ifdef CONFIG_HAVE_KVM_IRQCHIP
-	INIT_LIST_HEAD(&kvm->irq_routing);
 	INIT_HLIST_HEAD(&kvm->mask_notifier_list);
 #endif